pkg/auth/authenticator.go

```diff
@@ -145,6 +148,17 @@ func (a *Authenticator) Check(ctx context.Context, request *Request) (finalRespo
 	start_time := time.Now()
 	wsvc, ns, reason := readRequestContext(request)
 
+	// access logs
+	defer func() {
```

Comment:
In the most usages of loggers there's a convention that developers leverage log levels filtering to the logging library and most of the time there's not a condition to check print a log or not.
for example you always call logs but a configuration value sets the logging library to print the value or not.
This is a logrus example:
https://stackoverflow.com/questions/47514812/how-to-use-debug-log-in-golang

Reply:
Dear, It's not production ready yet. 

Just debugging and reason beyond this `if` clause is that current logging library used in Cerberus does not support `debug` level